src/content/concepts/module-federation.md

```diff
@@ -62,6 +64,8 @@ This plugin adds specific references to containers as externals and allows to im
 
 This plugin combines `ContainerPlugin` and `ContainerReferencePlugin`. Overrides and overridables are combined into a single list of specified shared modules.
 
+T> Learn more about how to configure ModuleFederationPlugin from [here](/plugins/module-federation-plugin)
+
```

Comment:
```suggestion
```
I would avoid tips whenever possible if they can be put in the body.

Reply:
Thank you for the suggestion!